CgfConverter/Structs/Structs.cs

```diff
@@ -442,4 +444,300 @@ public struct PhysicsStruct50
     public short Unknown22;
     public short Unknown23;
     public short Unknown24;
-}
\ No newline at end of file
+}
+
+public struct ShotInt3Quat
```

Comment:
Is this supposed to be `ShortInt3Quat`?

Reply:
This is sic from lumberyard. I think someone made a typo from their side but they just rolled with it.

Reply:
Ooof.  Makes me wonder what their PR process is like...